CHANGELOG.md

```diff
@@ -0,0 +1,450 @@
+All notable changes to this project will be documented in this file.
+We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
+
+## Unreleased
+
+### Added
+- **capital-framework:** Add atomic-component dependency to main package.json.
+
+### Changed
+- **cf-expandables:** Update atomic-component dependency to 1.3.2.
+- **cf-tables:** Update atomic-component dependency to 1.3.2.
+
+### Removed
+- **capital-framework:** All individual component lockfiles.
```

Comment:
This should have been "all components", rather than "capital-framework" in the original PR.

Reply:
Ah yesh